sklearn/multioutput.py

```diff
@@ -0,0 +1,214 @@
+"""
+MultiOneVsRestClassifier
+===========================
+This module includes several classes that extend base estimators to
+multi-target estimators.
+
+Most sklearn estimators use a response matrix to train a target function
+with a single output variable.  I.e. typical estimators use the
+training set X to estimate a target function f(X) that
+predicts a single Y.  The purpose of this class is to extend estimators
+to be able to estimate a series of target functions (f1,f2,f3...,fn)
+that are trained on a single X predictor matrix to predict a series
+of reponses (y1,y2,y3...,yn).
+"""
+
+# Author: Hugo Bowne-Anderson <hugobowne@gmail.com>
+# Author: Chris Rivera <chris.richard.rivera@gmail.com>
+# Author: Michael Williamson
+# Author: James Ashton Nichols <james.ashton.nichols@gmail.com>
+
+# License: BSD 3 clause
+
+import array
+import numpy as np
+import warnings
+import scipy.sparse as sp
+
+from .base import ClassifierMixin
+from .base import clone, is_classifier
+from .base import MetaEstimatorMixin, is_regressor
+from .preprocessing import LabelBinarizer
+from .metrics.pairwise import euclidean_distances
+from .utils import check_random_state
+from .utils.validation import _num_samples
+from .utils.validation import check_consistent_length
+from .utils.validation import check_is_fitted
+from .externals.joblib import Parallel
+from .externals.joblib import delayed
+from .externals import six
+
+from sklearn.multiclass import OneVsRestClassifier
+
+
+class MultiOneVsRestClassifier(ClassifierMixin):
```

Comment:
Yeah, I remember the reason discussed before. Missed this while switching between branches. Sorry.